index.js

```diff
@@ -0,0 +1,68 @@
+var play = prompt("What's your move?  --  (R)ock, (P)aper or (S)cissors");
+var x;
```

Comment:
Don't have vague variable names like x or y. This should be called something like playerChoice.